xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererDRMPRIME.cpp

```diff
@@ -23,7 +23,8 @@
 #include "cores/VideoPlayer/VideoRenderers/RenderCapture.h"
 #include "cores/VideoPlayer/VideoRenderers/RenderFactory.h"
 #include "utils/log.h"
-#include "windowing/gbm/DRMUtils.h"
+
+CWinSystemGbmGLESContext *CRendererDRMPRIME::m_pWinSystem = nullptr;
 
```

Comment:
We can init this in header file like the other member variables.

Reply:
No we can't :-)